### PR TITLE
Prepare for proper release

### PR DESCRIPTION
**Pull request by jsravn**
As we're using this in production now, I'm preparing it for a proper release.

* Update README appropriately
* Add CHANGELOG.md
* Update examples
* Modify Makefile to support manual release of tags, so we can do semantic versioning

### CODE REVIEW

**Comment by jsravn** [comment]
I'll tag it once this is merged.

**Comment by jsravn** [comment]
Should be good to go now :).

File: examples/feed-dns-demployment.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Example deployment for launching feed-dns, the route53 ingress controller.
```
Comment (sebbonnet) [inline examples/feed-dns-demployment.yml]: typo in filename

**Comment by jsravn** [comment]
@sebbonnet I think it's better to put usage at the top, so it's easy to find how to use it. I always find it annoying when a readme puts the usage info at the bottom somewhere. Once you already know what it does, the usage part is the most important.

There is already a CONTRIBUTING.md for contributions.

**Comment by jsravn** [comment]
I also think known limitations should be listed at the top, so it's easy to evaluate whether feed is usable.

**Comment by jsravn** [comment]
@sebbonnet Please have another look, I organised it a bit more closely to what you suggested. But I left usage at the top.

**Comment by sebbonnet** [comment]
@jsravn looks good - leaving you to squash and edit commit message

**Pull request merged**